node/resolvers/Queries/Organizations.ts

```diff
@@ -182,11 +187,13 @@ const Organizations = {
   getOrganizationsByEmail: async (
```

Comment:
I'm wondering if we should create another function called `getActiveOrganizationsByEmail`, so we don't change the behavior of this function... so the caller has 2 options:
1. call `getOrganizationsByEmail` to get all organizations for that email (either active or inactive).
2. call `getActiveOrganizationsByEmail` to get all/only active organizations for that email.

Reply:
sure, done